### ISSUE TITLE
Quorum config section is slightly misdesigned

### ISSUE THREAD

**Issue by AndrejMitrovic** [bosagora/agora#295]
Currently it's parsed as an array of Quorums, but in reality it should just be a quorum with a finite set of nested sub-quorums.

**Comment by MichaelKim20** [comment]
In SCP, innerSets contains information about the sub-quorums.
in XDR
```
struct SCPQuorumSet
{
    uint32 threshold;
    PublicKey validators<>;
    SCPQuorumSet innerSets<>;
};
```

**Comment by MichaelKim20** [comment]
I agree with @AndrejMitrovic

**Comment by Geod24** [comment]
Solved by #296
